make/autoconf/flags-cflags.m4

```diff
@@ -736,6 +736,13 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_CPU_DEP],
     fi
   fi
 
+  if test "x$TOOLCHAIN_TYPE" = xclang; then
+    if test "x$FLAGS_CPU" = xarm; then
+      $1_CFLAGS_CPU="-fsigned-char $ARM_ARCH_TYPE_FLAGS $ARM_FLOAT_TYPE_FLAGS -DJDK_ARCH_ABI_PROP_NAME='\"\$(JDK_ARCH_ABI_PROP_NAME)\"'"
+      $1_CFLAGS_CPU_JVM="-DARM"
```

Comment:
Are these flags that you'd want turned on for Linux if it was building using Clang on arm32?

Reply:
I don't know, there is the same thing for gcc just below: https://github.com/battleblow/openjdk-jdk11u/blob/bsd-port/make/autoconf/flags-cflags.m4#L751-L755
error without it:
```
ad_arm.cpp:19815:4: error: "ARM must be defined"
#  error "ARM must be defined"
   ^
1 error generated.
gmake[4]: *** [lib/CompileJvm.gmk:160: /usr/ports/java/openjdk11/work/openjdk-jdk11u-jdk-11.0.10-9-1/build/bsd-arm-normal-server-release/hotspot/variant-server/libjvm/objs/ad_arm.o] Error 1
gmake[4]: Leaving directory '/usr/ports/java/openjdk11/work/openjdk-jdk11u-jdk-11.0.10-9-1/make/hotspot'
```

the error for libsa is:
```
/usr/ports/java/openjdk11/work/openjdk-jdk11u-jdk-11.0.10-9-1/src/jdk.hotspot.agent/bsd/native/libsaproc/BsdDebuggerLocal.c:383:37: error: use of undeclared identifier 'NPRGREG'
  array = (*env)->NewLongArray(env, NPRGREG);
```

Reply:
Ok, linux ifdef Java_sun_jvm_hotspot_debugger_linux_LinuxDebuggerLocal_getThreadIntegerRegisterSet0 for supported arch:
https://github.com/battleblow/openjdk-jdk11u/blob/bsd-port/src/jdk.hotspot.agent/linux/native/libsaproc/LinuxDebuggerLocal.c#L399